test/integration/targets/setup_proxy/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+- name: install proxy.py
+  pip:
+    name: proxy.py
```

Comment:
It might be a good idea to pin it to the most recent release candidate (otherwise it may end up being pretty old). FYI.